utils/configure.js

```diff
@@ -14,21 +14,29 @@ function findClosestPackage() {
     return JSON.parse(file);
 }
 
-function isReactProject() {
+// Returns name of framework if one is used, false if not
+function isFrameworkProject() {
```

Comment:
Maybe we could refactor this into `resolveLibraryPrefix` and have it return `"default"`, instead of false if there are no special-case libraries found. Then we can get rid of the inline ternary operators later on and the name matches up a bit more nicely?

Reply:
Yeah good idea, added this in the new commit

Reply:
Thanks, looks good!